package.json

```diff
@@ -9,6 +9,12 @@
   },
   "type": "commonjs",
   "main": "dist/index",
+  "exports": {
+    ".": "./dist/index.js",
+    "./cache": "./dist/infra/cache/index.js",
+    "./config": "./dist/lib/config.js",
+    "./dist/*": "./dist/*"
```

Comment:
That's necessary so that deep imports from dist still work like before (eg for importing balena.sbvr or balena-init.sql or mocking the contracts in tests).